.github/workflows/ci.yml

```diff
@@ -32,6 +35,12 @@ jobs:
       SKIA_LIBRARY_DIR: "build/download"
 
     steps:
+    - name: "Set correct MB_PYTHON_VERSION for pypy"
+      run: |
+        if [ "pypy-3.7" == "${{ matrix.python-version }}" ]; then
+          echo "MB_PYTHON_VERSION=pypy3.6-7.3" >> $GITHUB_ENV
+          echo "TRAVIS_PYTHON_VERSION=pypy3.6-7.3" >> $GITHUB_ENV
```

Comment:
I think you need to modify the `actions/setup-python@v2` step so that it sets up a generic "3.x" python for the default Github Actions python instead of using `matrix.python-version`. Multibuild runs some python scripts (hence it requires some recent python) but it will download the requested python version (or activate it inside its docker image) so the python used to run multibuild's own scripts does not have to be, say, pypy3, or python3.6, etc.
We still want to parametrize the matrix based on `python-version` so that each time a different `MB_PYTHON_VERSION` is selected, but we want to use a generic python3 in GH actions/setup-python (whatever works with multibuild, it seems using pypy3 as the default GHA python in setup-python breaks some of its scripts..)

Reply:
Hrm. Ok, will look into matrices...

Reply:
That didn't help...